tests/monitoring/monitoring.go

```diff
@@ -134,49 +134,43 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 					},
 				},
 			}
-
-			Eventually(func() error {
-				kv, err = virtClient.KubeVirt(originalKv.Namespace).Update(context.Background(), kv, metav1.UpdateOptions{})
-				return err
-			}, 30*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+			patchBytes, err := patch.New(patch.WithAdd("/spec/customizeComponents", customizedComponents)).GeneratePayload()
+			Expect(err).ToNot(HaveOccurred())
+			_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
+			Expect(err).ToNot(HaveOccurred())
 
 			Eventually(func() string {
 				vh, err := virtClient.AppsV1().DaemonSets(originalKv.Namespace).Get(context.Background(), virtHandler.deploymentName, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				return vh.Spec.Template.Spec.NodeSelector[k8sv1.LabelHostname]
-			}, 90*time.Second, 5*time.Second).Should(Equal("does-not-exist"))
+			}).WithTimeout(90 * time.Second).WithPolling(5 * time.Second).Should(Equal("does-not-exist"))
 
 			Eventually(func() int {
 				vh, err := virtClient.AppsV1().DaemonSets(originalKv.Namespace).Get(context.Background(), virtHandler.deploymentName, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				return int(vh.Status.NumberAvailable + vh.Status.NumberUnavailable)
-			}, 90*time.Second, 5*time.Second).Should(Equal(0))
```

Comment:
in the codebase most often than not, the previous approach is used, why the change?

Reply:
This is something I'm trying to change. slowly...

Reply:
The Eventually/Constantly old API is not the best, and not consistent. I think we must move to the more explicit version of it.

Personally, I'm changing it whenever I can.